core/src/main/java/org/apache/accumulo/core/fate/Fate.java

```diff
@@ -393,6 +393,10 @@ public Fate(T environment, FateStore<T> store, boolean runDeadResCleaner,
     this.workFinder.start();
   }
 
+  protected long getDeadResCleanupDelay() {
```

Comment:
Could change this method name to include the unit `getDeadResCleanupDelayInSeconds` or make it return a duration.